oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/PropertyTest.java

```diff
@@ -274,4 +285,23 @@ public void testPropertyIteratorSize() throws Exception{
         assertEquals(3, pitr.getSize());
         assertEquals(3, Iterators.size(pitr));
     }
+    
+    public void testSetAndRemoveUnprotectedProperty() throws ValueFormatException, VersionException, LockException, ConstraintViolationException, RepositoryException {
```

Comment:
There is no need to declare `ValueFormatException`, `VersionException`, `LockException`, `ConstraintViolationException`. They are all `RepositoryException`.
```suggestion
    public void testSetAndRemoveUnprotectedProperty() throws RepositoryException {
```